plaidml/edsl/edsl.h

```diff
@@ -14,6 +14,8 @@
 #include "plaidml/core/core.h"
 #include "plaidml/edsl/ffi.h"
 
+#include "pmlc/util/enums.h"
```

Comment:
Sorry I didn't notice this before, but we can't do this here because users of the PlaidML API only need to depend on `libplaidml.so` and the `plaidml` header files. They won't have access to the `pmlc`, which is internal to the API.

Reply:
Removed this dependency.